src/main/java/net/dries007/tfc/objects/items/metal/ItemOreTFC.java

```diff
@@ -66,9 +67,18 @@ public ItemOreTFC(Ore ore)
             OreDictionaryHelper.register(this, "gem", ore);
             //noinspection ConstantConditions
             if (ore.getRegistryName().getPath().equals("lapis_lazuli"))
+            {
                 OreDictionaryHelper.register(this, "gem", "lapis");
+                OreDictionaryHelper.register(this, "ore", "lapis");
+            }
+            if (ore.getRegistryName().getPath().equals("cinnabar"))
+            {
+                OreDictionaryHelper.register(this, "ore", "cinnabar");
```

Comment:
Is this actually the only ore that should use `ore` instead of `gem`?

Reply:
It's not instead, but additionally. There may be others idk about it, which would need to investigate other mods for this.

Reply:
After a good inspection, i changed default minerals to include `ore` tag by default and the `gem` ones to also have `gemMineral` (ie: `gemKimberlite`)

Petrified wood tho is a confusing one. Is it an ore? The changes will made it to be called `orePetrifiedWood`, if that is right.